pkg/datapath/linux/route/route_linux.go

```diff
@@ -514,6 +514,24 @@ func lookupDefaultRoute(family int) (netlink.Route, error) {
 	return routes[0], nil
 }
 
+func DeleteRouteTable(table, family int) error {
+	var routeErr error
+
+	routes, err := netlink.RouteListFiltered(family, &netlink.Route{Table: table}, netlink.RT_FILTER_TABLE)
+	if err != nil {
+		return fmt.Errorf("Unable to list table %d routes: %s", table, err)
+	}
+
+	routeErr = nil
+	for _, route := range routes {
+		err := netlink.RouteDel(&route)
+		if err != nil {
+			routeErr = fmt.Errorf("Failed to delete route: %s", err)
```

Comment:
Only the last error encountered here will be returned, others will be lost. Maybe log and/or combine multiple errors?

Reply:
ok combining makes the most sense to me.

Reply:
It might print a bit clumsy if we get a bunch of errors here, but hopefully that doesn't happen.

~  routeErr = fmt.Errorf("%w: Failed to delete route: %s", routeErr, err)